advanced/sr_gui_biotac/src/sr_gui_biotac/biotac.py

```diff
@@ -267,15 +267,14 @@ def load_params(self):
         else:
             self.default_topic = ""
 
-    def __init__(self, context):
+    def __init__(self, context, rqt_plugin = True):
```

Comment:
At the beginning of the file, we should update the copyright to: "2013, 2019"

Reply:
Done